system-addon/content-src/components/Search/Search.jsx

```diff
@@ -24,7 +24,7 @@ class Search extends React.Component {
   onInputMount(input) {
     if (input) {
       this.controller = new ContentSearchUIController(input, input.parentNode,
-        "activity", "newtab");
+        "newtab", "newtab");
```

Comment:
Please add a comment here describing what this does, along with the side-effecti-ness that we've discovered, as well as what is likely to need to be changed if someone does need to touch this in the future.